### PR TITLE
Feature/daef 101 copy address to clipboard

### PR DESCRIPTION
**Pull request by tomothespian**
This PR adds introduces copy-to-clipboard icon notification message for copy wallet address feature on Receive page.

screen shot 2017-03-23 at 18 09 24

### CODE REVIEW

**Comment by DominikGuzei** [comment]
Reviewing this now

File: app/stores/WalletsStore.js [hunk 1]
```diff
@@ -262,4 +266,19 @@ export default class WalletsStore extends Store {
...
+    this.isWalletAddressCopyNotificationVisible = false;
+  }
+
+  _onShowWalletAddressCopyNotification = action(() => {
+    if (this._hideWalletAddressCopyNotificationTimeout) {
+      clearTimeout(this._hideWalletAddressCopyNotificationTimeout);
+    }
+    this._hideWalletAddressCopyNotificationTimeout = setTimeout(
+      this._hideWalletAddressCopyNotification,
+      10000
```
Comment (DominikGuzei) [inline app/stores/WalletsStore.js]: Please move this magic constant for hiding delay to a config value. I see that we don't have anything like that yet … so let's create `app/config.js` and add 
```js
export default {
  wallets: {
    ADDRESS_COPY_NOTIFICATION_DURATION: 10000,
  }
};
```

File: app/components/widgets/NotificationMessage.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    ]);
+
+    return (
+      <div className={notificationMessageStyles}>
+
+        {icon && <img className={styles.icon} src={icon} role="presentation" />}
+
+        {(
+          // eslint-disable-next-line react/no-danger
+          <span className={styles.message} dangerouslySetInnerHTML={{ __html: message }} />
```
Comment (tomothespian) [inline app/components/widgets/NotificationMessage.js]: @DominikGuzei yep - this is much better! Fixed in my latest commit ;)

File: app/stores/WalletsStore.js (hunk 1 above)
Comment (tomothespian) [inline app/stores/WalletsStore.js]: @DominikGuzei app/config.js added in my latest commit!

**Pull request merged**